src/ServiceInsight/App.xaml.cs

```diff
@@ -40,6 +41,7 @@ public partial class App : IAppCommands
     {
         public App()
         {
+            BlobCache.ApplicationName = "ServiceInsight";
```

Comment:
Does the cache has a schema? how does it cope with breaking changes? do we need to version it - by maybe by putting the version number in the cache name / storing folder? How do we guarantee backwards compatibility of the cache when we roll out a new version (since we're not strict on breaking changes).